src/server.js

```diff
@@ -166,18 +167,17 @@ function setup(config) {
 	return setupDao(config).then(() => setupExpress()).then(() => {
 		const port = process.env.PORT || 9000;
 		server = app.listen(port);
-		// eslint-disable-next-line no-console
-		console.log(`Server now listening on port ${port}`);
+		println(`Server now listening on port ${port}`);
 	});
 }
 
 /**
  * Stop the server
  */
 function stop() {
-	server.close();
-	// eslint-disable-next-line no-console
-	console.log('Server stopped');
+	server.close(() => {
```

Comment:
This technically seems a bit backwards to me. Doing DB teardown before server close, leaves greater opportunity for there to be http requests that are accepted but error out due to DB teardown. I would think it's better to stop accepting http requests, and then close the DB.

Reply:
Wait... I misread the code, it's the right way, hah. Clearly too sleepy.

Reply:
@Bluenaxela sleep! i'ts important! 🍊